File: programers_Lv1/예산.py
def solution(d, budget):
    answer = 0
    sum = 0

    d.sort()

    for i in d:
        sum += i
        if budget < sum:
            break
        answer += 1

    return answer

print(solution([1,3,2,5,4], 9))
print(solution(	[2, 2, 3, 3], 10))

File: programers_Lv1/예산.js
function solution(d, budget) {
  var answer = 0;
  let sum = 0;

  d.sort((a, b) => a - b);

  for (let i of d) {
    sum += i;
    if (budget < sum) {
      break;
    }
    answer++;
  }

  return answer;
}

console.log(solution([1, 3, 2, 5, 4], 9));
console.log(solution([2, 2, 3, 3], 10));
